Components/Avatar/Avatar.css

```diff
@@ -0,0 +1,33 @@
+/* -------------------- avatar----------------------- */
+
+/* ---------- profile avatar------------ */
+.avatar {
+    border-radius: 50%;
+    margin: 0 1rem;
+    object-fit: cover;
+    height: 9rem;
+    width: 9rem;
+    border: 2px solid var(--white);
+}
+
+
+/* -------------- sized avatar ---------- */
+.la {
```

Comment:
1. lg would be more appropriate than la.. 🤔 
2. A more general approach would be to use avatar-lg.  Only 'la' might create confusion. If you want to keep it like this, then you can try creating utility libraries 🤔  which would work on major components.

Reply:
I have changed the name to avatar-lg and respectively other also